tests/functional/13_cylc_play/suite.rc

```diff
@@ -0,0 +1,27 @@
+#!jinja2
```

Comment:
Shouldn't this file be named `flow.cylc`?

Reply:
🐆